src/Enterspeed.Source.UmbracoCms/Services/DataProperties/DefaultConverters/DefaultMultiUrlPickerPropertyValueConverter.cs

```diff
@@ -91,6 +93,8 @@ private ObjectEnterspeedProperty ConvertToEnterspeedProperty(Link link, IUmbraco
                 if (link.Udi.EntityType == "document")
                 {
                     var content = context.Content.GetById(link.Udi);
+                    // link.url always contains the url for the default culture (maybe a bug in Umbraco?), so we resolve the url for the correct culture our self
+                    url = content is not null ? content.GetUrl(_logger, culture, UrlMode.Absolute) : "#";
```

Comment:
You are right. This should have been resolved correctly before the method is called, since we are resolving the links with the culture param. :) Should we validate our approach and create a bug, if it is an issue?

Reply:
I have create a forum post on our.umbraco, but haven't got any responses: https://our.umbraco.com/forum/using-umbraco-and-getting-started//111666-multiurlpicker-does-not-return-culture-specific-urls